tests/monotouch-test/monotouch-test.sln

```diff
@@ -0,0 +1,46 @@
+﻿

```

Comment:
This file is unrelated to this PR and should be removed.

Reply:
Whoops, thanks for catching!

Reply:
@rolfbjarne Addressed!